src/decrypt.rs

```diff
@@ -61,6 +62,26 @@ pub(crate) async fn prepare_decryption(
     from: &str,
     message_time: i64,
 ) -> Result<DecryptionInfo> {
+    if mail.headers.get_header(HeaderDef::ListPost).is_some() {
+        if mail.headers.get_header(HeaderDef::Autocrypt).is_some() {
+            warn!(
```

Comment:
if this is expected behaviour it should be info, not warn

Reply:
It's not expected in the greater scheme of things, the mailing list software should remove the Autocrypt header, because they will leak the original sender's email (at least @adbenitez does this in his mailing lists; if you have a mailing list where you keep the original From address anyway, then it's fine).

But we could 1. still make it an info, since it's not a DC problem, or 2. add a note "That's unexpected, the mailing list software should remove autocrypt headers for privacy."

Reply:
yeah, i think i'm still in favour of moving this down to info...